docs/activation.md

```diff
@@ -0,0 +1,34 @@
+# The `activation` ping
+
+## Description
+This ping is intended to provide a measure of the activation of mobile products.
+
+## Scheduling
+The `activation` ping is automatically sent at startup, after Glean is initialized, and contains
```

Comment:
```suggestion
The `activation` ping is automatically sent at first startup, after Glean is initialized, and contains
```

Reply:
We should make this clear this ping is sent only once, only during the very first startup (or during subsequent startups if it hasn't been sent yet?).